framework_tvb/tvb/interfaces/web/static/js/genericTVB.js

```diff
@@ -133,6 +133,11 @@ function fireOnClick(redirectElem) {
     }
 }
 
+// ---------- Function for rendering HTML elements with Mathjax
+function renderWithMathjax(element, elementToAppend){
+    element.append(elementToAppend);
```

Comment:
Maybe Add a flag in this function's signature append=true/false and thus avoid the .empty() calls before calling this function?